### PR TITLE
ErrorManagement

### PR DESCRIPTION
**Pull request by IonutPad15**

### CODE REVIEW

File: IssueTracker/IssueTrackerAPI/Controllers/CommentController.cs [hunk 1]
```diff
@@ -37,17 +37,31 @@ public async Task<IResult> GetById(int id)
...
-        commentResponse.Attachements = await AutoMapperConfig.GetAttachements(comment.Attachements);
-        return Results.Ok(commentResponse);
+        try
+        {
+            commentResponse.Attachements = await AutoMapperConfig.GetAttachements(comment.Attachements);
+            return Results.Ok(commentResponse);
+        }
+        catch (FileSystemException ex)
+        {
+            return Results.Conflict(ex.Message);
```
Comment (iiusan) [inline IssueTracker/IssueTrackerAPI/Controllers/CommentController.cs]: Conflict should only be used for add/edit operations.

File: IssueTracker/IssueTrackerAPI/Controllers/FileController.cs [hunk 2]
```diff
@@ -63,18 +61,33 @@ public async Task<IResult> PostFile([FromForm] IFormFile formFile, [FromForm] in
     public async Task<IActionResult> GetFiles(IEnumerable<FileGetRequest> fileName)
     {
         var fileAttachment = _mapper.Map<IEnumerable<IssueTracker.FileSystem.Models.File>>(fileName);
-        var response = await _fileProvider.GetAsync(fileAttachment);
-        if (response == null) throw new ArgumentException("Result is null!");
-        return Ok(response);
+        try
+        {
+            var response = await _fileProvider.GetAsync(fileAttachment);
+            if (response == null) return BadRequest("there are no foles");
```
Comment (iiusan) [inline IssueTracker/IssueTrackerAPI/Controllers/FileController.cs]: typo

File: IssueTracker/IssueTrackerAPI/Controllers/CommentController.cs (hunk 1 above)
Comment (IonutPad15) [reply IssueTracker/IssueTrackerAPI/Controllers/CommentController.cs]: ok, I'll change

File: IssueTracker/IssueTrackerAPI/Controllers/FileController.cs (hunk 2 above)
Comment (IonutPad15) [reply IssueTracker/IssueTrackerAPI/Controllers/FileController.cs]: will change

**Review by iiusan — APPROVED** [review]

**Pull request merged**